components/BottomNavigation/examples/BottomNavigationExplicitlySetColorExample.swift

```diff
@@ -15,11 +15,14 @@
 import Foundation
 import MaterialComponents.MaterialButtons
 import MaterialComponents.MaterialPalettes
-import MaterialComponents.MaterialBottomNavigation_ColorThemer
 
 class BottomNavigationExplicitlySetColorExample: UIViewController {
 
-  @objc var colorScheme = MDCSemanticColorScheme()
+  let containerScheme: MDCContainerScheming = {
+    let containerScheme = MDCContainerScheme()
+    containerScheme.colorScheme = MDCSemanticColorScheme()
```

Comment:
Tip: I'd prefer if the default container scheme values were kept.

Reply:
Done.